src/Catel.Core/Catel.Core.Shared/Data/ModelBase.propertyediting.cs

```diff
@@ -21,6 +22,18 @@ public partial class ModelBase
         /// <param name="name">Name of the property.</param>
         /// <param name="value">Value of the property.</param>
         /// <param name="notifyOnChange">If <c>true</c>, the <see cref="INotifyPropertyChanged.PropertyChanged"/> event will be invoked.</param>
+        [NotifyPropertyChangedInvocator]
```

Comment:
I'm not sure if this should be used here. It's more than just "NotifyPropertyChangedInvocator", that's the actual base class doing that.